include/bouncer.h

```diff
@@ -75,6 +75,12 @@ enum PauseMode {
 	P_SUSPEND = 2		/* wait for buffers to be empty */
 };
 
+enum ShutDownMode {
+	SHUTDOWN_NONE = 0,
+	SHUTDOWN_WAIT_FOR_SERVERS = 1,
+	SHUTDOWN_IMMEDIATE = 2,
+};
```

Comment:
Do you need the numbers?

Reply:
The 0 one is necessary, so we can do `if (cf_shutdown) { ... }`. So I left that one and removed the others, because those were indeed not needed.

Reply:
There is no harm specifying the `= 0` but the spec says it starts with zero if not specified [1]. Since you mention this `if`:

```
if (cf_shutdown != SHUTDOWN_NONE) {
...
```

is my preference.

[1] https://en.cppreference.com/w/c/language/enum

Reply:
>There is no harm specifying the = 0 but the spec says it starts with zero if not specified [1]

Hmm, interesting I didn't know that. I got in the habit of at least providing the 0 one because @petere suggested that in a few PRs.